### PR TITLE
Remove depreciated list() function.

### LINKED ISSUE #89: Not compatible with terraform 0.15
When using the module with terraform 0.15.0, I get the following error:
```
│ Error: Error in function call
│ 
│   on .terraform/modules/log_export_audit_rapid7_pubsub/main.tf line 27, in locals:
│   27:   log_sink_writer_identity = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.writer_identity, list("")), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.writer_identity, list("")), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.writer_identity, list("")), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.writer_identity, list("")), 0) : ""
│ 
│ Call to function "list" failed: the "list" function was deprecated in Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to write a literal list.
```

`list` is deprecated as explained here: https://www.terraform.io/upgrade-guides/0-15.html#legacy-configuration-language-features

### PR DESCRIPTION
**Pull request by grugnog**
This should address the Terraform 1.15 compatibility issue described in, fixes #89

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: main.tf [hunk 1]
```diff
@@ -24,13 +24,13 @@ locals {
...
-  log_sink_parent_id       = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.project, list("")), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.folder, list("")), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.org_id, list("")), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.billing_account, list("")), 0) : ""
+  log_sink_writer_identity = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.writer_identity, []), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.writer_identity, []), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.writer_identity, []), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.writer_identity, []), 0) : ""
```
Comment (morgante) [inline main.tf:27]: The intention is to have a list of one empty string, so this is better:
```suggestion
  log_sink_writer_identity = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.writer_identity, []), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.writer_identity, []), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.writer_identity, []), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.writer_identity, [""]), 0) : ""
```

**Review by morgante — CHANGES_REQUESTED** [review]

**Comment by grugnog** [comment]
@morgante updated, thanks!

**Review by morgante — APPROVED** [review]
Thanks!

**Pull request merged**